ccmlib/scylla_repository.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import with_statement
+
+import os
+import tarfile
+import tempfile
+import time
+import shutil
+import glob
+
+from six import print_
+from six.moves import urllib
+
+from ccmlib.common import (ArgumentError, CCMError, get_default_path, rmdirs, validate_install_dir)
+from ccmlib.repository import __download
+
+GIT_REPO = "http://github.com/scylladb/scylla.git"
+
+RELOCATABLE_URLS_BASE = 'https://s3.amazonaws.com/downloads.scylladb.com/relocatable/{0}/{1}'
```

Comment:
that will not be the path for the official versions it will be probably be parallel to the scylla releases
http://downloads.scylladb.com/downloads/scylla/
if we will have one relocatable package it will be parallel to rpm/deb - if we will have one for each distribution it will be under rpm/deb

Reply:
o.k. we could amend this, once we have at least one formal release with relocatable.